google/cloud/security/inventory/pipelines/base_pipeline.py

```diff
@@ -85,7 +85,7 @@ def _to_int(value):
         # pylint: disable=no-else-return
         if value is None:
             return None
-        elif value == '':
+        elif not value:
```

Comment:
Would it make sense to change lines 86-91 to:

```
if not value:
  return None
else:
  return int(value)
```

Also I wonder if we should wrap in a try/except, in case the int conversion fails. Thoughts?